chat.html

```diff
@@ -80,6 +93,7 @@
             messageContainer.appendChild(messageTextElement);
             messagesContainer.appendChild(messageContainer);
             messagesContainer.scrollTop = messagesContainer.scrollHeight;
```

Comment:
This line shouldn't be needed anymore as it's the one which tries to scroll

Reply:
i removed it

Reply:
looks good to me